src/components/view/view-context-menu-builder.ts

```diff
@@ -142,4 +144,28 @@ export class ViewEventContextMenuBuilder {
         };
     }
 
+    getHeaderToggleContextMenu(enabledColumns: Map<string, boolean>) {
```

Comment:
I don't think this should take any arguments. This class is already storing the `uiStore`, it should just read the current columns and set them directly. That also means we don't need the `onHeaderColumnOptionChange` constructor argument either.